geotrellis-python/geotrellis/python/edited_avro/avro_builder.py

```diff
@@ -0,0 +1,239 @@
+from __future__ import absolute_import
```

Comment:
There is an issue in jira (https://issues.apache.org/jira/browse/AVRO-1690), the avro team didn't accept the PR and didn't comment.